src/blocksmith/components/link-modal/index.js

```diff
@@ -20,13 +20,15 @@ export default function LinkModal(props) {
   const { selectedLink } = props;
   const [url, setUrl] = useState(selectedLink ? selectedLink.url : '');
   const [target, setTarget] = useState(selectedLink && selectedLink.target);
+  const [noFollow, setNoFollow] = useState(!!(selectedLink && selectedLink.rel && selectedLink.rel.includes('nofollow')));
```

Comment:
@joshlevy you can shorthand this big time
```suggestion
  const [noFollow, setNoFollow] = useState(!!selectedLink?.rel?.includes?.('nofollow'));
```

(probably dont even really need to cast the bool but no harm in it)